db/db.js

```diff
@@ -0,0 +1,156 @@
+import {
+  Sequelize,
+  STRING,
+  BOOLEAN,
+  INTEGER,
+  TEXT,
+  DATE,
+} from "sequelize";
+
+const sequelize = new Sequelize("multiuser_blog", "root", "root", {
+    host: "localhost",
+    dialect: "mysql",
+    port: 8889,
+    define: {
+        timestamps: true,
+    },
+});
+
+const Users = sequelize.define("users", {
+  user_id: {
```

Comment:
no need for user_id, as sequelize automatically creates id as primary key. do same for the others remove them all.

eg:
const Categories = sequelize.define("categories", {
    category_name: {
      type: STRING,
      allowNull: false,
    },
    category_description: {
      type: STRING,
      allowNull: true,
    },
  });
  
  this is ok.